autoarray/operators/transformer.py

```diff
@@ -112,3 +117,200 @@ def transformed_mapping_matrices_from_mapping_matrix(self, mapping_matrix):
         )
 
         return [real_transformed_mapping_matrix, imag_transformed_mapping_matrix]
+
+
+class TransformerFFT(object):
+    def __init__(self, uv_wavelengths, grid):
+
+        super(TransformerFFT, self).__init__()
+
+        self.uv_wavelengths = uv_wavelengths.astype("float")
+        self.grid = grid.in_1d_binned
+
+        self.u_fft = np.fft.fftshift(
+            np.fft.fftfreq(
+                grid.shape_2d[0], grid.pixel_scales[0] * units.arcsec.to(units.rad)
+            )
+        )
+        self.v_fft = np.fft.fftshift(
+            np.fft.fftfreq(
+                grid.shape_2d[1], grid.pixel_scales[1] * units.arcsec.to(units.rad)
+            )
+        )
+        u_fft_meshgrid, v_fft_meshgrid = np.meshgrid(self.u_fft, self.v_fft)
+
+        # ... This is nessesary due to the way the grid in autolens is set up.
+        self.shift = np.exp(
+            -2.0
+            * np.pi
+            * 1j
+            * (
+                self.grid.pixel_scales[0]
+                / 2.0
+                * units.arcsec.to(units.rad)
+                * u_fft_meshgrid
+                + self.grid.pixel_scales[0]
+                / 2.0
+                * units.arcsec.to(units.rad)
+                * v_fft_meshgrid
+            )
+        )
+
+        self.uv = np.array(
```

Comment:
I wonder if array is happy to accept an iterable meaning you don't need to cast to list?